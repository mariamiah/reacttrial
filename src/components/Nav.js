import React from 'react';
import { Link, Route, BrowserRouter as Router  } from 'react-router-dom';


class Nav extends React.Component{
    render(){
        return(
    
            <div>
                <header>
                    <li><Link to="/home">Home</Link></li>
                    <li><Link to="/about">About</Link></li>
                    <li><Link to="/articles">Articles</Link></li>  
                </header>
            </div>

        )
    }

}
export default Nav;