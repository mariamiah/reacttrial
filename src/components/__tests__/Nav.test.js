import React from 'react';
import Nav from '../Nav';
import { MemoryRouter } from 'react-router';
import renderer from 'react-test-renderer';



test('should render without crushing', () => {
    const component = renderer.create(
      <MemoryRouter>
        <Nav />
      </MemoryRouter>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });