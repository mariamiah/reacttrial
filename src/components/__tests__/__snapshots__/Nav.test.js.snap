// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crushing 1`] = `
<div>
  <header>
    <li>
      <a
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </li>
    <li>
      <a
        href="/articles"
        onClick={[Function]}
      >
        Articles
      </a>
    </li>
  </header>
</div>
`;
